docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -99,16 +99,20 @@ stored on the data node according to the type of data node:
 
 It is also possible to partially read the contents of data nodes, which comes in handy when dealing with large amounts
 of data.
-This can be achieved using the `DataNode.filter()^` method:
+This can be achieved by providing an operator, a Tuple of (field_name, value, comparison operator),
+or a list of operators to the `DataNode.filter()^` method:
 
 ```python linenums="1"
-data_node.filter([('field_name_like_temperature', 14, Operator.EQUAL), ('field_name_like_temperature', 10, Operator.EQUAL)], JoinOperator.OR))
+data_node.filter([('field_name', 14, Operator.EQUAL), ('field_name', 10, Operator.EQUAL)], JoinOperator.OR))
```

Comment:
Double-quotes should be preferred over single-quotes.
  You are more likely to have strings (delimited by double-quotes) containing single-quote character than the other way aroud.